EIPS/eip-3675.md

```diff
@@ -175,6 +180,12 @@ After the upgrade of the consensus mechanism only the beacon chain network will
 
 It is recommended for the client software to not propagate descendants of any terminal PoW block to reduce the load on processing the P2P component and stop operating in the environment with unknown security properties.
 
+### Allowing for transition difficulty changes
+
+Considering [network anomalies and/or attack scenarios](#ability-to-jump-between-terminal-pow-blocks), it may be necessary to quickly coordinate on a change in the transition point. 
```

Comment:
suggest moving the link becuase the extent of  "attack scenarios" isn't really described in that section. In particular, one thing this helps fight against is the event of consistent 51% attacks/reorgs that prevent getting to terminal pow difficulty.
```suggestion
Considering [network anomalies](#ability-to-jump-between-terminal-pow-blocks) and/or attack scenarios, it may be necessary to quickly coordinate on a change in the transition point. 
```

Reply:
fair enough!